### DISCUSSION TITLE
I Can't Even Get The Launcher Open

### DISCUSSION THREAD

**Discussion by StallionsPanther** [X2CommunityCore/xcom2-launcher#256] [Help]
![Screenshot (12)](https://user-images.githubusercontent.com/76641128/103165149-de16ce80-47c8-11eb-99cf-9df4d36cf67b.png)

![Screenshot (13)](https://user-images.githubusercontent.com/76641128/103165132-bfb0d300-47c8-11eb-9641-7d8c29da904e.png)

I think this instructional text applies to my problem:
"Dialog "Please start steam first!" appears
Your Antivirus tool may prevent AML from accessing the Steam API. Especially "Comodo Cloud Antivirus" has been known to cause issues.
The Steamworks documentation gives 3 reasons why this might happen:
The Steam client isn't running. A running Steam client is required to provide implementations of the various Steamworks interfaces.
The Steam client couldn't determine the AppID of the game. Make sure you have steam_appid.txt in your game directory.
Your application is not running under the same user context as the Steam client, including admin privileges.
In other words make sure that:
Steam is running
the file steam_appid.txt exists in the AML folder
neither (or both) of Steam and AML are running with admin privileges"

The problem is, I still don't know what to do after reading this. I've been trying for hours to make sure the "file steam_appid.txt exists in the AML folder" but I have no idea how to do that. I've opened the pathway with my AVG Antivirus. Steam is not run in admin and I don't know how to check if AML is running in admin; I assume not. I've browsed the forums and cannot find a solution. Please help.

**Comment by RevZero** [comment]
When you start AML for the first time, it will try to create the `steam_appid.txt` (the contained game id depends on the target game you selected, either "XCOM 2" or "XCOM Chimera Squad"). 

The first screenshot indicates, that AML was unable to create the file. It says "Access denied", that can have different reasons. The straightforward reason would be, that the current windows user just does not have permission to write to that folder. Judging from the path, "Swaghard" is the Windows profile you are using? Or do you log in as a different user?

Check if you can manually create a text file or something in that folder. Try extracting AML to a different folder. If you can not figure it out, check our discord channel and ask for help there.
